<launch>


<!-- Launching USB CAM launch files and Dynamixel controllers -->
    <arg name="device" default="/dev/video0" />

    <node name="uvc_camera_node" pkg="uvc_camera" type="uvc_camera_node" output="screen">
    	<remap from="/image_raw" to="/usb_cam/image_raw" />
        <param name="device" value="$(arg device)" />
        <param name="width" value="320" />
        <param name="height" value="240" />
        <param name="frame_rate" value="20" />
        <param name="exposure" value="0" />
        <param name="gain" value="100" />  
    </node>

<!-- Starting face tracker node  -->
   <rosparam file="$(find face_shooter)/config/track.yaml" command="load"/>

   <node name="face_tracker" pkg="face_shooter" type="face_tracker_node.py" output="screen" />

</launch>
